Source/WebKit/NetworkProcess/NetworkSessionCreationParameters.cpp

```diff
@@ -117,7 +117,7 @@ std::optional<NetworkSessionCreationParameters> NetworkSessionCreationParameters
     if (!sessionID)
         return std::nullopt;
     
-    std::optional<std::optional<UUID>> dataStoreIdentifier;
+    std::optional<Markable<UUID>> dataStoreIdentifier;
```

Comment:
We can just decode it into `std::optional`, and then we can assign it to markable.

Reply:
This is more correct.  I really ought to just make it NetworkSessionCreationParameters.serialization.in which would generate this, but that should probably be a separate PR.